src/chartkit/components/Pie.js

```diff
@@ -46,8 +48,19 @@ class Pie extends Component {
 
   onMouseEnter(data, e) {
     if (data) {
-      const { index } = data;
-      this.setState({ highlightedIndex: index });
+      const { index, indexValue } = data;
+      const label = {
+        highlightedIndex: index,
```

Comment:
these values log as `{"highlightedIndex":1}` which do not give us as much information as 
```js
  const label = {[id]: value}

```
which logs as ` {"Trio": 1680}`

The goal here is to keep these as human readable to make the most sense in downstream analysis

Reply:
Changed to:
```javascript
const { id, index, value } = data;
this.setState({ highlightedIndex: index, highlightedValue: value });
if (this.props.analyticsTracking) {
  trackUserInteraction({
    category: this.props.analyticsTracking.category,
    action: `Chart Pie : ${TRACKING_EVENTS.actions.hover}`,
    label: `${id}: ${value}`,
  })
}
```